opentelemetry-example/src/main/scala/zio/telemetry/opentelemetry/example/http/BackendApp.scala

```diff
@@ -4,18 +4,33 @@ import io.opentelemetry.api.trace.SpanKind
 import io.opentelemetry.api.trace.propagation.W3CTraceContextPropagator
 import io.opentelemetry.context.propagation.{ TextMapGetter, TextMapPropagator }
 import zio.telemetry.opentelemetry.Tracing
-import zio.telemetry.opentelemetry.TracingSyntax._
 import zio.telemetry.opentelemetry.example.http.{ Status => ServiceStatus }
 import zhttp.http.{ !!, ->, /, Headers, Http, HttpApp, Method, Response }
 import zio.json.EncoderOps
-import zio.ZIO
+import zio.{ URLayer, ZIO, ZLayer }
+import BackendApp._
 
 import java.lang
 import scala.jdk.CollectionConverters._
 
+final case class BackendApp(tracing: Tracing) {
+
+  val routes: HttpApp[Any, Throwable] =
+    Http.collectZIO { case request @ Method.GET -> !! / "status" =>
+      tracing.spanFrom(propagator, request.headers, getter, "/status", SpanKind.SERVER, Map.empty[Throwable, Nothing]) {
```

Comment:
Hm, requirement to add this `Map.empty[Throwable, Nothing]` doesn't seem to be that convenient for clients. Maybe we should add the default value of `Map.empty` to all `toErrorStatus` parameters in `Tracing`? :thinking:

Reply:
I'm going to find a better encoding for syntax, so this will be changed to `response.spanFrom` where `toErrorStatus` would have a default value as well as we have for all syntax methods. But, of course, we might want to add a default value for the `Tracing` methods. Hope it makes sense.

Reply:
Right after I wrote the previous comment, I realized that it makes sense to avoid having syntax at all since using the module pattern 2.0 approach leads us to have two options on how to call the methods:
- `tracing.spanFrom(...)(effect)`
- `effect.spanFrom(...)(tracing)`
which doesn't make any difference.

Reply:
I'll add default values for this and the rest methods in PR that I'm working on right now https://github.com/zio/zio-telemetry/pull/586, so this may remain as it is at the moment.